### PR TITLE
fix: pass profile to status hint; handle customized param names

### PR DESCRIPTION
**Pull request by kakhaUrigashvili**
fixing 2 minor issues
1) passing profile to status hint if it was provided for the command
2) fixing to pass correct parameter name if it was customized. for example: stageV2 should be state.

### CODE REVIEW

File: lib/commands/smapi/smapi-command-handler.js [hunk 1]
```diff
@@ -115,8 +116,13 @@ const _showCheckStatusHint = (locationHeader) => {
         const baseCliCommand = processor.processOperationName(operationName);
         const queryStringParameters = queryString.parse(queryStringPart);
         parameters = { ...swaggerMatch.params, ...queryStringParameters };
-        const cliParams = Object.keys(parameters).map(key => `--${kebabCase(key)} ${parameters[key]}`).join(' ');
-        statusCheckHintCommand = `ask smapi ${baseCliCommand} ${cliParams}`;
+        const cliParams = Object.keys(parameters).map(key => {
+            // mapping custom command option if exists
+            const commandOption = kebabCase(apiToCommanderMap.get(key) || key);
```
Comment (kakhaUrigashvili) [inline lib/commands/smapi/smapi-command-handler.js:121]: main use case. if parameter is called stageV2 it is called stage in ask cli
https://github.com/alexa/ask-cli/blob/develop/lib/commands/smapi/customizations/parameters.json#L3

File: lib/commands/smapi/smapi-command-handler.js [hunk 2]
```diff
@@ -97,7 +98,7 @@ const _matchSwaggerPattern = (url, swaggerPatternUrl) => {
     return { swaggerPatternUrl, params };
 };
 
-const _showCheckStatusHint = (locationHeader) => {
+const _showCheckStatusHint = (locationHeader, profile) => {
```
Comment (kakhaUrigashvili) [inline lib/commands/smapi/smapi-command-handler.js:101]: passing down the profile to show in the hint command if it was provided

File: lib/commands/smapi/smapi-command-handler.js [hunk 3]
```diff
@@ -115,8 +116,13 @@ const _showCheckStatusHint = (locationHeader) => {
         const baseCliCommand = processor.processOperationName(operationName);
         const queryStringParameters = queryString.parse(queryStringPart);
         parameters = { ...swaggerMatch.params, ...queryStringParameters };
-        const cliParams = Object.keys(parameters).map(key => `--${kebabCase(key)} ${parameters[key]}`).join(' ');
-        statusCheckHintCommand = `ask smapi ${baseCliCommand} ${cliParams}`;
+        const cliParams = Object.keys(parameters).map(key => {
+            // mapping custom command option if exists
+            const commandOption = kebabCase(apiToCommanderMap.get(key) || key);
+            return `--${commandOption} ${parameters[key]}`;
```
Comment (RonWang) [inline lib/commands/smapi/smapi-command-handler.js:122]: Looking through these customization cases, so far there are three types: `skipUnwrap`, `name`, `skip`, and the `customParameters`. Based on this use case, I believe `skip` should also be considered, i.e. the vendorId should be prevented to be displayed in the hint string.

Comment (RonWang) [reply lib/commands/smapi/smapi-command-handler.js:122]: Using current latest version, I can verify the vendorId is still part of the string based on the following test:
```
$ ask smapi submit-skill-for-certification -s amzn1.ask.skill.12345 --publication-method MANUAL_PUBLISHING -p pub
[Warn]: This is an asynchronous operation. Check the progress using the following command: ask smapi list-skills-for-vendor --vendor-id MWD3O0I5T2G52 --skill-id amzn1.ask.skill.12345
Command executed successfully!
```


Please also add a test for this case to make sure similar case can be handled well.

**Pull request merged**